notes_app/notes/management/commands/testdata.py

```diff
@@ -8,23 +8,20 @@
 
 def gen_random_users(user_count):
     """Puts a set of random users of given length into the db."""
-    username_tpl = "demo_user_{:0>2}"
-    first_name = "demo"
-    last_name = "demo"
-    email_tpl = "demo{}@example.com"
-    password = "demo"
+    username_tpl = 'demo_user_{:0>2}'
+    first_name = 'demo'
+    last_name = 'demo'
+    email_tpl = 'demo{}@example.com'
+    password = 'demo'
     for i in xrange(user_count):
-        try:
-            User.objects.get(username=username_tpl.format(i))
-        except:
-            user = User.objects.create(
-                    username=username_tpl.format(i),
-                    password=make_password(password),
-                    first_name=first_name,
-                    last_name=last_name,
-                    email=email_tpl.format(i),
-                    is_active=True
-                )
+        (user, is_created) = \
```

Comment:
format nit: `\` is ugly, http://python.net/~goodger/projects/pycon/2007/idiomatic/handout.html#long-lines-continuations
it could be like
```
(user, is_created) = User.objects.get_or_create(
    username=username_tpl.format(i))
```